packages/protobuf-encoder/types.js

```diff
@@ -0,0 +1,33 @@
+import { Root, Type, Field, Enum } from 'protobufjs/light'
+
+let MessageType = new Enum('Type', {
+  no_type: 0,
+  welcome: 1,
+  disconnect: 2,
+  ping: 3,
+  confirm_subscription: 4,
+  reject_subscription: 5
+})
+
+let Command = new Enum('Command', {
+  unknown_command: 0,
+  subscribe: 1,
+  unsubscribe: 2,
+  message: 3
+})
+
+let Message = new Type('Message').add(
+  new Field('type', 1, 'Type'),
+  new Field('command', 2, 'Command'),
+  new Field('identifier', 3, 'string'),
+  new Field('data', 4, 'string'),
+  new Field('message', 5, 'bytes'),
```

Comment:
Here is a thing I forgot to mention (though it's present in the [docs](https://docs.anycable.io/anycable-go/binary_formats?id=protobuf)) that `Message` field contains a Msgpack-encoded object ('cause it could be anything).

Yeah, we're mixing Protobuf with Msgpack.

Reply:
@palkan FYI I added tests on it in the last commit